CMakeLists.txt

```diff
@@ -256,83 +288,328 @@ if(BUILD_JS)
     install(DIRECTORY services/user/AdminSys/ui/dist/ DESTINATION ${CMAKE_INSTALL_DATADIR}/psibase/services/admin-sys COMPONENT ServerData)
     install(DIRECTORY services/user/CommonSys/common/fonts DESTINATION ${CMAKE_INSTALL_DATADIR}/psibase/services/admin-sys/common COMPONENT ServerData)
     install(FILES services/user/CommonSys/common/rpc.mjs services/user/CommonSys/common/keyConversions.mjs DESTINATION ${CMAKE_INSTALL_DATADIR}/psibase/services/admin-sys/common COMPONENT ServerData)
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/share/psibase/services/admin-sys/packages DESTINATION ${CMAKE_INSTALL_DATADIR}/psibase/services/admin-sys COMPONENT ServerData)
 endif()
 
+# Build services
+include(pack_service.cmake)
+
+set(SERVICE_DIR ${CMAKE_CURRENT_BINARY_DIR}/share/psibase/packages)
+
+psibase_package(
```

Comment:
Could each of these be added into each project directory CMakeLists?

Reply:
It's possible, but
- The js build would also need to be moved which increases build system recursion depth
- Some of the packages (DocSys, Default) don't have services
- In some cases, several services should be included in a single package, e.g. TokenSys and SymbolSys probably belong in the same package because they're mutually dependent.